packages/amplify-cli/src/input-manager.ts

```diff
@@ -235,7 +236,12 @@ export function verifyInput(pluginPlatform: PluginPlatform, input: CLIInput): In
         commandString += ' ' + input.subCommands!.join(' ');
       }
 
-      result.message = `The Amplify CLI can NOT find command: ${commandString}`;
+      const packageManager = (await getPackageManager())?.packageManager ?? 'npm';
+      const executeCommand = packageManager === 'npm' ? 'npx' : `${packageManager} dlx`;
+
+      const amplifyGen2Message = `For Amplify Gen 2, install the @aws-amplify/backend-cli package or execute using the package name directly:${EOL}${executeCommand} @aws-amplify/backend-cli amplify${commandString}`;
```

Comment:
Should the command just be `${executeCommand} @aws-amplify/backend-cli ${commandString}`? (ie no "amplify")?

Reply:
Whoops, fixed in [10b679c](https://github.com/aws-amplify/amplify-cli/pull/13708/commits/10b679c22b8b1152a3bf9b27a095564b78b919f0)